add process hazard event queue and put it to celery

meomancer opened the pull request:
Fix the #1 of https://github.com/kartoza/geocris-inasafe-fba/issues/21

meomancer committed:
add process hazard event queue and put it to celery

meomancer committed:
fix if style fixture is not found

meomancer committed:
remove console

meomancer committed:
fix command for resetting hazard event queue

lucernae commented:
fix #26

lucernae commented on mapserver/django/django_project/mapserver/tasks.py:
```diff
@@ -0,0 +1,12 @@
+logger = logging.getLogger(__name__)
+
+
+@shared_task(name='mapserver.process_hazard_event_queue')
```
not associated with mapserver. This is just a generic task. Let's use `fba` namespace.

lucernae commented on mapserver/docker-compose.yml:
```diff
@@ -4,23 +4,53 @@ services:
     image: camptocamp/mapserver:7.4
     restart: unless-stopped
 
+  rabbitmq:
```
Do not use rabbitmq for now.
Use in memory broker for celery.
This is because we want to stay as lightweight as possible for the generic case.
GeoCRIS deployment would not allow us to freely install more service.

lucernae commented on mapserver/docker-compose.yml:
```diff
@@ -4,23 +4,53 @@ services:
+      - RABBITMQ_NODENAME=rabbit
+
+  worker:
+    build: django
```
Make a settings file to configure environment where it is not possible to have separate worker instance (django and celery stays in the same server).

lucernae reviewed (commented):
Some review as notes

meomancer committed:
add django test for celery peformance

meomancer commented:
> add django test for celery peformance

lucernae committed:
Refactor django apps

- Separate generic FbA Apps into it's own module
- Refactor celery settings
- Refactor docker-compose settings
- Add more development command to Makefile for worker
- Do not auto restart worker

lucernae committed:
Merge branch 'master' of github.com:kartoza/geocris-inasafe-fba into django-celery

lucernae committed:
Merge branch 'master' of github.com:kartoza/geocris-inasafe-fba into django-celery
